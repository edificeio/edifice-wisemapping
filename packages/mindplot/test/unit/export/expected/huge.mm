<map version="1.0.1">
    <node ID="ID_null" TEXT="I Care">
        <edge COLOR="#121110"/>
        <node ID="ID_null" POSITION="left" STYLE="bubble" TEXT="De markt">
            <richcontent TYPE="NOTE">
                <html xmlns="http://www.w3.org/1999/xhtml">
                    <head><head/>
                    <body>
                        <p/>
                    </body>
                </html>
            </richcontent>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="hygi&#235;ne"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diensten"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="energie">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kamer herkent gebruiker"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="automatische lichten"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="inrichting (woning)">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="advies">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kleuren en stijlen"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eigendomsverificatie"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit"/>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="medicijnen">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieven"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="neveneffecten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="werking"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="electronica">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="productie verloop">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="testfasen"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wat/wanneer"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="door wie"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="levensloop">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="gebruikscycli"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="reparatieservice"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="rechtstreekse link naar producent bij falen"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eigendomsverificatie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="afdanking (sluikstorten)"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="koop-verkoop"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="GPS tracking"/>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="voeding">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="zie bij handelingen --&gt;winkelen"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="rectangle" TEXT="kleding">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="stijlen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="begeleiding bij het winkelen naar gelijkaardige kleding"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <richcontent TYPE="NOTE">
                        <html xmlns="http://www.w3.org/1999/xhtml">
                            <head><head/>
                            <body>
                                <p/>
                            </body>
                        </html>
                    </richcontent>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteitsinfo bij aankoop">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="reviews"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="info uit cloud"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="informatiebronnen per merk">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="slimme kledij"/>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="veiligheid">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="criminaliteit">
                <richcontent TYPE="NOTE">
                    <html xmlns="http://www.w3.org/1999/xhtml">
                        <head><head/>
                        <body>
                            <p/>
                        </body>
                    </html>
                </richcontent>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Aantal ongevallen/criminele feiten registreren">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Terugkoppeling naar politie/verkeersdienst"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Meer controle"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Mapping van probleemsituaties"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diefstal">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="tracking via gps"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wapenhandel">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opsporen"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="drugs">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opsporen">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="testen per woonregio (via afvoerstelsels)"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="individuele tests"/>
                    </node>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="verkeer">
                <richcontent TYPE="NOTE">
                    <html xmlns="http://www.w3.org/1999/xhtml">
                        <head><head/>
                        <body>
                            <p/>
                        </body>
                    </html>
                </richcontent>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="ongevallen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="info over de staat van het wegdek">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aanpassing van max. snelheid"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="waarschuwingen"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wegenwerken"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="oorzaak">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opsporen van vluchtmisdrijf"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Automatisch opbellen van hulpdiensten"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="nagaan van de oorzaak"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="preventie">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bestraffen van chauffeurs?"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="roekeloos rijgedrag opsporen">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="via sensoren aan boord versnelling, manouvres, snelheid,... registreren"/>
                        </node>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Openbare fietsen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Fiets bestellen in grote steden"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Trackingsysteem, # fietsen op 1 plaats"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Openbaar vervoer"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="filevorming">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Omleiding zoeken"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="preventie">
                        <richcontent TYPE="NOTE">
                            <html xmlns="http://www.w3.org/1999/xhtml">
                                <head><head/>
                                <body>
                                    <p/>
                                </body>
                            </html>
                        </richcontent>
                    </node>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="handelingen">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="winkelen">
                <font BOLD="true" SIZE="12"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="winkelhulp">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="boodschappenlijst">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieven indien uitverkocht"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="dichtste bij bovenaan"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="recepten generator">
                        <richcontent TYPE="NOTE">
                            <html xmlns="http://www.w3.org/1999/xhtml">
                                <head><head/>
                                <body>
                                    <p/>
                                </body>
                            </html>
                        </richcontent>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bestellingen op afstand"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="digitale portemonnee"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="productinfo">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieve producten"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="di&#235;et checker"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="voedingstoffen"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bio"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="databases v. reviews"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="via cloud"/>
                        </node>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="prijs"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="allergie">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Salmonella sensor"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Sensor voor vers fruit en vlees"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Gezondheidscontrole"/>
                        </node>
                    </node>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="industrie"/>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aandoeningen en situaties">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="dementen">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wassen"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wegloopdetectie"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="zwangerschap">
                <font BOLD="true" SIZE="12"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="geboortetimer">
                    <richcontent TYPE="NOTE">
                        <html xmlns="http://www.w3.org/1999/xhtml">
                            <head><head/>
                            <body>
                                <p/>
                            </body>
                        </html>
                    </richcontent>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="info over voeding">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="via cloud info over voeding">
                        <richcontent TYPE="NOTE">
                            <html xmlns="http://www.w3.org/1999/xhtml">
                                <head><head/>
                                <body>
                                    <p/>
                                </body>
                            </html>
                        </richcontent>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wat is gezond"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="baby monitoren"/>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="sport">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opvolging van de sporter">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="gps"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="cadans"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="energieverbruik"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bloeddruk"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="hartslag"/>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="cultuur">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="festival"/>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="bouw">
            <richcontent TYPE="NOTE">
                <html xmlns="http://www.w3.org/1999/xhtml">
                    <head><head/>
                    <body>
                        <p/>
                    </body>
                </html>
            </richcontent>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opvolging zoals plannen zijn getekend">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="nameten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="juistheid van materialen"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="isolatie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="warmtemetingen"/>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="ziektes">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="stress">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="in welke afdeling een probleem">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aanpak"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aan welke factoren ligt dat"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="afreageren"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="thuisverzorging">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="link naar ziekenhuis"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="link naar dokter"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="meting van de symptomen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="objectieve metingen"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="toevoeging van eigen waarneming"/>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="verschillende ziektes">
                <richcontent TYPE="NOTE">
                    <html xmlns="http://www.w3.org/1999/xhtml">
                        <head><head/>
                        <body>
                            <p/>
                        </body>
                    </html>
                </richcontent>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="anorexia">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eten ze voldoende"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="nagaan of ze eten"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="astma">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="detectie van luchtkwaliteit">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="in kaart brengen van">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="allergie veroorzakende deeltjes"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="pollen"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="vervuiling"/>
                        </node>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="epilepsie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aanval voorspellen??"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diabetes">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="suikerspiegel meten">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="automatische inspuiting"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alert indien te laag"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aids">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="vergroten van de database aan info"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="mucoviscidose">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alert voor donor"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="longcapaciteit meten"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="MS">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bevorderen van communicatie tijdens de aftakeling">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar familie en vrienden toe"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar de dokters toe"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="feedback van de lichaamsconditie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aan dokter">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="op afstand consulatie"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="volledig overzicht"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="contacteert patient indien nodig"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="link">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar behandelend arts"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar hulpdiensten"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar ziekenhuis"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aan de pati&#235;nt">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="waarschuwingen"/>
                    </node>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="milieu">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieve energie">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kernenergie"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="zon"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wind"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="waterreserves">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="vervuiling"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="reserves"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eco systemen">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diversiteit"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="dioxines"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="afval">
                <node ID="ID_null" POSITION="left" STYLE="fork"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="cradle 2 cradle"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="riolen"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="verwerking"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="pollutie"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="mag ik dit door mijn gootsteen kappen"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bosbouw">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bodemvervuiling">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="onderzoek"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="parasieten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="ziektes bij bomen"/>
            </node>
        </node>
    </node>
</map>