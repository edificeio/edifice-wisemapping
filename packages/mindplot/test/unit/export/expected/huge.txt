1 I Care
	1.1 De markt
		* Note: 
		1.1.1 hygiëne
		1.1.2 diensten
		1.1.3 energie
			1.1.3.1 kamer herkent gebruiker
			1.1.3.2 automatische lichten
		1.1.4 inrichting (woning)
			1.1.4.1 advies
				1.1.4.1.1 kleuren en stijlen
			1.1.4.2 monitoring
				1.1.4.2.1 eigendomsverificatie
				1.1.4.2.2 kwaliteit
		1.1.5 medicijnen
			1.1.5.1 alternatieven
			1.1.5.2 neveneffecten
			1.1.5.3 werking
			1.1.5.4 kwaliteit
		1.1.6 electronica
			1.1.6.1 monitoring
				1.1.6.1.1 productie verloop
					1.1.6.1.1.1 testfasen
					1.1.6.1.1.2 wat/wanneer
					1.1.6.1.1.3 door wie
				1.1.6.1.2 levensloop
					1.1.6.1.2.1 gebruikscycli
				1.1.6.1.3 kwaliteit
					1.1.6.1.3.1 reparatieservice
					1.1.6.1.3.2 rechtstreekse link naar producent bij falen
			1.1.6.2 eigendomsverificatie
				1.1.6.2.1 afdanking (sluikstorten)
				1.1.6.2.2 koop-verkoop
				1.1.6.2.3 GPS tracking
		1.1.7 voeding
			1.1.7.1 zie bij handelingen -->winkelen
		1.1.8 kleding
			1.1.8.1 stijlen
				1.1.8.1.1 begeleiding bij het winkelen naar gelijkaardige kleding
			1.1.8.2 monitoring
				* Note: 
			1.1.8.3 kwaliteitsinfo bij aankoop
				1.1.8.3.1 reviews
				1.1.8.3.2 info uit cloud
			1.1.8.4 informatiebronnen per merk
				1.1.8.4.1 slimme kledij
	1.2 veiligheid
		1.2.1 criminaliteit
			* Note: 
			1.2.1.1 Aantal ongevallen/criminele feiten registreren
				1.2.1.1.1 Terugkoppeling naar politie/verkeersdienst
				1.2.1.1.2 Meer controle
				1.2.1.1.3 Mapping van probleemsituaties
			1.2.1.2 diefstal
				1.2.1.2.1 tracking via gps
			1.2.1.3 wapenhandel
				1.2.1.3.1 opsporen
			1.2.1.4 drugs
				1.2.1.4.1 opsporen
					1.2.1.4.1.1 testen per woonregio (via afvoerstelsels)
					1.2.1.4.1.2 individuele tests
		1.2.2 verkeer
			* Note: 
			1.2.2.1 ongevallen
				1.2.2.1.1 info over de staat van het wegdek
					1.2.2.1.1.1 aanpassing van max. snelheid
					1.2.2.1.1.2 waarschuwingen
					1.2.2.1.1.3 wegenwerken
				1.2.2.1.2 oorzaak
					1.2.2.1.2.1 opsporen van vluchtmisdrijf
					1.2.2.1.2.2 Automatisch opbellen van hulpdiensten
					1.2.2.1.2.3 nagaan van de oorzaak
				1.2.2.1.3 preventie
					1.2.2.1.3.1 bestraffen van chauffeurs?
					1.2.2.1.3.2 roekeloos rijgedrag opsporen
						1.2.2.1.3.2.1 via sensoren aan boord versnelling, manouvres, snelheid,... registreren
			1.2.2.2 Openbare fietsen
				1.2.2.2.1 Fiets bestellen in grote steden
				1.2.2.2.2 Trackingsysteem, # fietsen op 1 plaats
			1.2.2.3 Openbaar vervoer
			1.2.2.4 filevorming
				1.2.2.4.1 Omleiding zoeken
				1.2.2.4.2 preventie
					* Note: 
	1.3 handelingen
		1.3.1 winkelen
			1.3.1.1 winkelhulp
				1.3.1.1.1 boodschappenlijst
					1.3.1.1.1.1 alternatieven indien uitverkocht
					1.3.1.1.1.2 dichtste bij bovenaan
				1.3.1.1.2 recepten generator
					* Note: 
				1.3.1.1.3 bestellingen op afstand
				1.3.1.1.4 digitale portemonnee
				1.3.1.1.5 productinfo
					1.3.1.1.5.1 alternatieve producten
					1.3.1.1.5.2 diëet checker
					1.3.1.1.5.3 kwaliteit
						1.3.1.1.5.3.1 voedingstoffen
						1.3.1.1.5.3.2 bio
						1.3.1.1.5.3.3 databases v. reviews
						1.3.1.1.5.3.4 via cloud
					1.3.1.1.5.4 prijs
					1.3.1.1.5.5 allergie
						1.3.1.1.5.5.1 Salmonella sensor
						1.3.1.1.5.5.2 Sensor voor vers fruit en vlees
						1.3.1.1.5.5.3 Gezondheidscontrole
	1.4 industrie
	1.5 aandoeningen en situaties
		1.5.1 dementen
			1.5.1.1 eten
			1.5.1.2 wassen
			1.5.1.3 wegloopdetectie
		1.5.2 zwangerschap
			1.5.2.1 geboortetimer
				* Note: 
			1.5.2.2 info over voeding
				1.5.2.2.1 via cloud info over voeding
					* Note: 
				1.5.2.2.2 wat is gezond
			1.5.2.3 baby monitoren
	1.6 sport
		1.6.1 opvolging van de sporter
			1.6.1.1 gps
			1.6.1.2 cadans
			1.6.1.3 energieverbruik
			1.6.1.4 bloeddruk
			1.6.1.5 hartslag
	1.7 cultuur
		1.7.1 festival
	1.8 bouw
		* Note: 
		1.8.1 opvolging zoals plannen zijn getekend
			1.8.1.1 nameten
			1.8.1.2 juistheid van materialen
			1.8.1.3 isolatie
				1.8.1.3.1 warmtemetingen
	1.9 ziektes
		1.9.1 stress
			1.9.1.1 monitoring
				1.9.1.1.1 in welke afdeling een probleem
					1.9.1.1.1.1 aanpak
					1.9.1.1.1.2 aan welke factoren ligt dat
			1.9.1.2 afreageren
		1.9.2 thuisverzorging
			1.9.2.1 link naar ziekenhuis
			1.9.2.2 link naar dokter
			1.9.2.3 meting van de symptomen
				1.9.2.3.1 objectieve metingen
				1.9.2.3.2 toevoeging van eigen waarneming
		1.9.3 verschillende ziektes
			* Note: 
			1.9.3.1 anorexia
				1.9.3.1.1 eten ze voldoende
				1.9.3.1.2 nagaan of ze eten
			1.9.3.2 astma
				1.9.3.2.1 detectie van luchtkwaliteit
					1.9.3.2.1.1 in kaart brengen van
						1.9.3.2.1.1.1 allergie veroorzakende deeltjes
						1.9.3.2.1.1.2 pollen
						1.9.3.2.1.1.3 vervuiling
			1.9.3.3 epilepsie
				1.9.3.3.1 aanval voorspellen??
			1.9.3.4 diabetes
				1.9.3.4.1 suikerspiegel meten
					1.9.3.4.1.1 automatische inspuiting
					1.9.3.4.1.2 alert indien te laag
			1.9.3.5 aids
				1.9.3.5.1 vergroten van de database aan info
			1.9.3.6 mucoviscidose
				1.9.3.6.1 alert voor donor
				1.9.3.6.2 longcapaciteit meten
			1.9.3.7 MS
				1.9.3.7.1 bevorderen van communicatie tijdens de aftakeling
					1.9.3.7.1.1 naar familie en vrienden toe
					1.9.3.7.1.2 naar de dokters toe
			1.9.3.8 feedback van de lichaamsconditie
				1.9.3.8.1 aan dokter
					1.9.3.8.1.1 op afstand consulatie
					1.9.3.8.1.2 volledig overzicht
					1.9.3.8.1.3 contacteert patient indien nodig
				1.9.3.8.2 link
					1.9.3.8.2.1 naar behandelend arts
					1.9.3.8.2.2 naar hulpdiensten
					1.9.3.8.2.3 naar ziekenhuis
				1.9.3.8.3 aan de patiënt
					1.9.3.8.3.1 waarschuwingen
	1.10 milieu
		1.10.1 alternatieve energie
			1.10.1.1 kernenergie
			1.10.1.2 zon
			1.10.1.3 wind
		1.10.2 waterreserves
			1.10.2.1 vervuiling
			1.10.2.2 reserves
		1.10.3 eco systemen
			1.10.3.1 diversiteit
		1.10.4 dioxines
		1.10.5 afval
			1.10.5.1 
			1.10.5.2 cradle 2 cradle
			1.10.5.3 riolen
			1.10.5.4 verwerking
		1.10.6 pollutie
		1.10.7 mag ik dit door mijn gootsteen kappen
		1.10.8 bosbouw
			1.10.8.1 bodemvervuiling
				1.10.8.1.1 onderzoek
			1.10.8.2 parasieten
			1.10.8.3 ziektes bij bomen
